README.md

```diff
@@ -799,3 +801,42 @@ curl "localhost:8080/api/v1/requests" \
 In the above case, Cachito would fetch the submodules `cachito-pip-with-deps`, `cachito-npm-test` and
 then process them as a regular pip and npm package respectively.
 
+### yarn
+
+Cachito handles the yarn package manager in much the same way as the [npm](###npm) package manager.
```

Comment:
```suggestion
Cachito handles the yarn package manager in much the same way as the [npm](#npm) package manager.
```
See the difference between
https://github.com/akhmanova/cachito/tree/yarn-documentation###npm
https://github.com/akhmanova/cachito/tree/yarn-documentation#npm